src/main/java/com/artipie/asto/s3/S3Storage.java

```diff
@@ -386,69 +280,49 @@ public S3Storage(final S3AsyncClient client, final String bucket, final boolean
     }
 
     /**
-     * Uploads content using multipart upload.
+     * Save multipart.
      *
-     * @param key Object key.
-     * @param content Object content to be uploaded.
-     * @return Completion stage which is completed when upload is completed or aborted.
+     * @param updated The estimated content.
+     * @param key The key.
```

Comment:
@andreoss I think it would be better to get a more detailed description of params

Reply:
@genryxy Done.